src/main/api/2/studio-api-2.yaml

```diff
@@ -1456,6 +1462,208 @@ paths:
         '500':
           $ref: '#/components/responses/InternalServerError'
 
+  /content:
+    get:
+      tags:
+        - content
+      summary: content
+      operationId: content
+      responses:
+        '200':
+          description: OK
+  /content/types:
+    get:
+      tags:
+        - content type
+      summary: Get content types
+      operationId: getContentTypes
+      parameters:
+        - name: siteId
+          in: query
+          description: Site ID
+          required: false
+          schema: 
+            type: string
+        - name: class
+          in: query
+          description: Content type class - e.g. `page`, `component`, `taxonomy` e.t.c.
+          required: false
+          schema:
+            type: string
+        - name: path
+          in: query
+          description: Target path where content type will be used
```

Comment:
Not really understand what is this path for, this API returns all content types, why is the path needed?